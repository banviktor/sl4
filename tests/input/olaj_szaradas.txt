randomization off
questions off
start_game 2
2 3
3 1
toggle_oil
jump
jump
N
list_smudges
jump
jump
N
list_smudges
jump
jump
N
list_smudges
jump
jump
N
list_smudges
jump
jump
N
list_smudges
jump
jump
N
list_smudges
exit
