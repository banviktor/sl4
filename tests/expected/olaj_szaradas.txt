Véletlenszerű események kikapcsolva.
Kérdezés kikapcsolva.
Pálya betöltve.
Új játék elkezdve 2 játékossal.
Ez a(z) 1. kör.
A Piros Robot következik.
Olajfolt be.
Ragacs ki.
Olajfolt lehelyezve.
Új pozíció: 2.0, 3.0
Ez a(z) 1. kör.
A Sárga Robot következik.
Új pozíció: 3.0, 1.0
Ez a(z) 2. kör.
A Piros Robot következik.
Foltok listája:
	Olaj: 2.0, 3.0
Új pozíció: 2.0, 3.0
Ez a(z) 2. kör.
A Sárga Robot következik.
Új pozíció: 3.0, 1.0
Ez a(z) 3. kör.
A Piros Robot következik.
Foltok listája:
	Olaj: 2.0, 3.0
Új pozíció: 2.0, 3.0
Ez a(z) 3. kör.
A Sárga Robot következik.
Új pozíció: 3.0, 1.0
Ez a(z) 4. kör.
A Piros Robot következik.
Foltok listája:
	Olaj: 2.0, 3.0
Új pozíció: 2.0, 3.0
Ez a(z) 4. kör.
A Sárga Robot következik.
Új pozíció: 3.0, 1.0
Ez a(z) 5. kör.
A Piros Robot következik.
Foltok listája:
	Olaj: 2.0, 3.0
Új pozíció: 2.0, 3.0
Ez a(z) 5. kör.
A Sárga Robot következik.
Új pozíció: 3.0, 1.0
Ez a(z) 6. kör.
A Piros Robot következik.
Foltok listája:
	Olaj: 2.0, 3.0
Új pozíció: 2.0, 3.0
Ez a(z) 6. kör.
A Sárga Robot következik.
Új pozíció: 3.0, 1.0
Ez a(z) 7. kör.
A Piros Robot következik.
Foltok listája:
